packages/reactivity/src/reactive.ts

```diff
@@ -232,24 +306,83 @@ export function isShallow(value: unknown): boolean {
   return !!(value && (value as Target)[ReactiveFlags.IS_SHALLOW])
 }
 
+/**
+ * Checks if an object is a proxy created by `reactive` or `readonly`.
+ *
+ * @param value The value to check.
+ */
 export function isProxy(value: unknown): boolean {
   return isReactive(value) || isReadonly(value)
 }
 
+/**
+ * Returns the raw, original object of a reactive or readonly proxy.
```

Comment:
Let's update to the latest version of the docs (https://vuejs.org/api/reactivity-advanced.html#toraw)